system-test/fixtures/sample/src/index.js

```diff
@@ -20,7 +20,7 @@
 const webrisk = require('@google-cloud/web-risk');
 
 function main() {
-  const webRiskServiceV1Beta1Client = new webrisk.v1beta1.WebRiskServiceV1Beta1Client();
+  const webRiskServiceV1Beta1Client = new webrisk.WebRiskServiceV1Beta1Client();
```

Comment:
`index.ts` is no longer exporting beta version client [PR](https://github.com/googleapis/nodejs-web-risk/pull/160/files#diff-c02e9b813b281ba213bf25affbd4fcc9L23)
it should not omit the `v1beta1` here